packages/theme/src/adaptation-index.less

```
@@ -1,37 +1,37 @@
 @import './button/adaptation.less';
 @import './button-group/adaptation.less';
-// @import './link/adaptation.less';
+@import './link/responsive.less';
 // @import './action-menu/adaptation.less';
```

Stylelint (17.4.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)